crypto/CMakeLists.txt

```diff
@@ -38,9 +38,39 @@ if (CONFIG_NRF_CC310_BL)
   target_link_libraries(${IMAGE}nrfxlib_crypto INTERFACE ${CC310_BL_LIB})
 endif()
 
+if (CONFIG_NRF_CC310_PLATFORM)
+  set (NRF_CC310_PLATFORM_BASE ${CMAKE_CURRENT_SOURCE_DIR}/nrf_cc310_platform)
+  set (NRF_CC310_PLATFORM_VER 0.9.0)
+  if (NOT CONFIG_CC310_INTERRUPTS_BACKEND)
+    set(CC310_PLATFORM_FLAVOR no-interrupts)
+  endif()
+  set(NRF_CC310_PLATFORM_LIB
+    ${NRF_CC310_PLATFORM_BASE}/${lib_path}/${CC310_PLATFORM_FLAVOR}/libnrf_cc310_platform_${NRF_CC310_PLATFORM_VER}.a
+  )
+
+  if (NOT EXISTS ${NRF_CC310_PLATFORM_LIB})
+    message(WARNING "This combination of SoC, floating point ABI, and interrupts settings is not supported by the nrf_cc310_platform lib."
+                    "(${NRF_CC310_PLATFORM_LIB} doesn't exist.)")
+  endif()
+  #
+  # Uses a normal add_library to avoid --whole-archive
+  #
+  add_library(${IMAGE}platform_cc310 STATIC IMPORTED GLOBAL)
+  set_target_properties(${IMAGE}platform_cc310 PROPERTIES IMPORTED_LOCATION
+      "${NRF_CC310_PLATFORM_LIB}")
+  target_include_directories(${IMAGE}platform_cc310 INTERFACE
+      "${NRF_CC310_PLATFORM_BASE}/include")
+  zephyr_link_libraries(${IMAGE}platform_cc310)
+  #
+  # Add companion sources to directly to zephyr
+  #
+  zephyr_sources(${NRF_CC310_PLATFORM_BASE}/src/nrf_cc310_platform_abort_zephyr.c)
+  zephyr_sources(${NRF_CC310_PLATFORM_BASE}/src/nrf_cc310_platform_mutex_zephyr.c)
+endif()
+
 if (CONFIG_CC310_BACKEND)
   set(NRF_CC310_BASE ${CMAKE_CURRENT_SOURCE_DIR}/nrf_cc310_mbedcrypto)
-  set(NRF_CC310_VER 0.8.1)
+  set(NRF_CC310_VER 0.9.0)
```

Comment:
as this PR has not yet been approved, I really think we should post fix with `rcX` so that we can better track changes to the lib, and ensure that the correct version of the library is indeed being used.

If all libs between each force push is named `0.9.0` it's hard to differentiate between them.
Also using `rcX` will establish a workflow similar to how releases are done in rest of NCS.

Could be similar to: https://github.com/NordicPlayground/nrfxlib/blob/b84c83a0ead833e12d5b493f898d27ed4f0c459d/crypto/CMakeLists.txt#L43

Reply:
This comment also applies to the file names, unfortunately, github does not allow for review comments on bin files

Reply:
Will work with the process on upcoming versions. We had some late-breaking changes that we didn't forsee in late stages of creating this lib (no functional changes except library dependency)